rllib/examples/cartpole_lstm.py

```diff
@@ -39,6 +39,9 @@
             "env": StatelessCartPole,
             "model": {
                 "use_lstm": True,
+                # Switch on time-major (and with that the usage of
+                # traj. view API).
+                "_time_major": args.torch,
```

Comment:
Can we make this the default without needing to set it separately (basically, remove the time_major arg)?

Reply:
ok, makes sense. If `use_lstm` -> time_major=True (implicitly).

Reply:
done.